test/simple/f.d.ts

```diff
@@ -0,0 +1 @@
+export const foo: 'bar';
```

Comment:
A more valid `.d.ts` file wouldn't have values, remember 😄 It'd be `export const foo: string;`